packages/@core/src/kwil-actions/create-kwil-client.ts

```diff
@@ -1,51 +1,65 @@
-import { type KwilSigner, NodeKwil, Utils, WebKwil } from "@kwilteam/kwil-js";
-import type { ActionBody } from "@kwilteam/kwil-js/dist/core/action";
+import { type KwilSigner, NodeKwil, WebKwil } from "@kwilteam/kwil-js";
+import type { ActionBody, CallBody, PositionalParams } from "@kwilteam/kwil-js/dist/core/action";
+import type { ValueType } from "@kwilteam/kwil-js/dist/utils/types";
 import invariant from "tiny-invariant";
+import { actionSchema } from "./schema";
 
 interface CreateKwilClientParams {
   chainId?: string;
-  dbId?: string;
   nodeUrl: string;
 }
 
 interface KwilActionReqParams {
   name: string;
-  description?: string;
```

Comment:
Having a description is a must in Júlio's eyes. Is there any way we can get this back?

Reply:
It is [there](https://github.com/idos-network/idos-sdk-js/pull/584/files#diff-93c7228d9a480b1f1e0e7f333ec5cbbd92128af47d84713d74aa4a21ec1e73daR20)

Reply:
It is only for mutations (which is right)

Reply:
Ah, I had missed it, perfect! :ok_hand: